src/basic/sysctl-util.c

```diff
@@ -44,7 +44,7 @@ char *sysctl_normalize(char *s) {
         return s;
 }
 
-int sysctl_write(const char *property, const char *value) {
+int sysctl_write(const char *property, const char *value, Hashmap *cache) {
```

Comment:
"cache" is a misnomer for the concept though, no? It's a "shadow", not a "cache".